domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/gtfs/trips/Trip.java

```diff
@@ -0,0 +1,330 @@
+/*
+ * Copyright (c) 2020. MobilityData IO.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.domain.entity.gtfs.trips;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.mobilitydata.gtfsvalidator.domain.entity.gtfs.BikesAllowedStatus;
+import org.mobilitydata.gtfsvalidator.domain.entity.gtfs.EntityBuildResult;
+import org.mobilitydata.gtfsvalidator.domain.entity.gtfs.GtfsEntity;
+import org.mobilitydata.gtfsvalidator.domain.entity.gtfs.WheelchairAccessibleStatus;
+import org.mobilitydata.gtfsvalidator.domain.entity.notice.base.Notice;
+import org.mobilitydata.gtfsvalidator.domain.entity.notice.error.MissingRequiredValueNotice;
+import org.mobilitydata.gtfsvalidator.domain.entity.notice.error.UnexpectedEnumValueNotice;
+
+import java.util.ArrayList;
+
+/**
+ * Class for all entities defined in trips.txt. Can not be directly instantiated: user must use the
+ * {@link TripBuilder} to create this.
+ */
+public class Trip extends GtfsEntity {
+    @NotNull
+    private final String routeId;
+    @NotNull
+    private final String serviceId;
+    @NotNull
+    private final String tripId;
+    @Nullable
+    private final String tripHeadsign;
+    @Nullable
+    private final String tripShortName;
+    @Nullable
+    private final DirectionId directionId;
+    @Nullable
+    private final String blockId;
+    @Nullable
+    private final String shapeId;
+    private final WheelchairAccessibleStatus wheelchairAccessibleStatus;
```

Comment:
Seems like nullability annotations are missing on the last two members

Reply:
Opened an issue on this matter. See https://github.com/MobilityData/gtfs-validator/issues/226